papergres.go

```diff
@@ -407,20 +407,31 @@ func (db *Database) GenerateInsert(obj interface{}) *Query {
 	return db.Schema("public").GenerateInsert(obj)
 }
 
-// Insert inserts the passed in object
+// Insert inserts the passed in object in DB.
```

Comment:
If you have comments spanning more than one line then I believe a multiline comment would be a better solution. It will make your code less messy and easily readable.

Reply:
For documenting code in Go, it is considered idiomatic to always use line comments`//` syntax. You would only use the block comments `/* ... */` syntax for debugging and they appear mostly as package comments. This approach makes them format well when extracted into godoc documentation.

For reference: Check out the comments in the `io` package - https://cs.opensource.google/go/go/+/refs/tags/go1.17:src/io/io.go
Also, feel free to read along for more details:  https://golang.org/doc/effective_go#commentary